plugin/db/mysql/mysql.go

```diff
@@ -1019,6 +1019,9 @@ func flushTablesWithReadLock(ctx context.Context, conn *sql.Conn, database strin
 
 	var tableNames []string
 	for _, table := range tables {
+		if table.TableType != baseTableType {
```

Comment:
Could you please add a comment about the reason why skipping views? Thanks.

Reply:
LG otherwise.

Reply:
I think it's quite obvious that we're looking for tables from the the function name flushTablesWithReadLock.

Reply:
SG.